packages/libs/react-ui/src/components/Tooltip/Tooltip.css.ts

```diff
@@ -2,82 +2,86 @@ import { sprinkles } from '@theme/sprinkles.css';
 import { vars } from '@theme/vars.css';
 import { style, styleVariants } from '@vanilla-extract/css';
 
-export const container = style([
-  sprinkles({
-    placeItems: 'center',
-    gap: '$2',
-    borderRadius: '$md',
-    paddingX: '$4',
-    paddingY: '$3',
-    border: 'none',
-    fontSize: '$base',
-    backgroundColor: '$neutral1',
-    color: '$neutral6',
-    width: 'max-content',
-    position: 'fixed',
-    display: 'none',
-    pointerEvents: 'none',
-  }),
-  {
-    zIndex: 10,
-    top: '50%',
-    marginRight: '-50%',
-    border: `${vars.borderWidths.$md} solid ${vars.colors.$neutral2}`,
-  },
-]);
-
-const baseArrow = style([
+export const base = style([
   sprinkles({
     position: 'absolute',
-    width: '$4',
-    height: '$4',
-    backgroundColor: '$neutral1',
+    backgroundColor: '$neutral6',
+    fontSize: '$sm',
+    paddingY: '$xs',
+    paddingX: '$sm',
+    borderRadius: '$md',
+    color: '$neutral1',
     pointerEvents: 'none',
+    width: 'max-content',
+    maxWidth: '$maxContentWidth',
   }),
   {
-    rotate: '45deg',
+    ':before': {
+      content: '',
+      position: 'absolute',
+      borderTop: '6px solid transparent',
```

Comment:
I guess 6px is not a defined size in our tokens?

Reply:
No it is not, but also I don't think this one should be a token as it is very specific and we don't want it to change based on the browser fontsize

Reply:
Alright, thanks for clarifying 👌